dom-querying/main.js

```diff
@@ -0,0 +1,8 @@
+console.log(document.querySelector('h1'));
```

Comment:
I would rewrite lines 1 - 3 as: 
```
var $heading = document.querySelector('h1');

console.log('$heading', $heading);
console.dir($heading);
```

The benefit is that you are saving the location of the h1 in memory and not doing a second query to find it.

This wouldn't be a performance hit on a page this small but would be an issue on a larger site.

Please follow this example for the rest of the code reducing the amount of queries.

Also, on line 2 you were just logging the string `'h1'` instead of a value associated with the `h1` tag.